### ISSUE TITLE
yolov4x-mish.cfg does not converge

### ISSUE THREAD

**Issue by dalhorde** [AlexeyAB/darknet#7424]
I am using yolov4 for a month now and I noticed something strange. 
When I train my model with different cfg files I have the same issue : the loss does not converge (yolov4x-mish) or begins to converge and then becomes nan (yolov4-sam-mish). This is quite strange because the network does not change and I try to keep the same parameters and hyperparameters. Nevertheless when I follow AlexeyAB's instructions for custom dataset it works well. 
Do you have any ideas ? I tried to modify the number of subdivisions, random (works better with random=0 but it is far more slower, width and height)
I have a custom dataset with only one class. 
Here is my cfg file for yolov4x-mish : 

[yolov4x-mish.txt](https://github.com/AlexeyAB/darknet/files/6035927/yolov4x-mish.txt)

Thank you for your information 
![chart_yolov4x-mish](https://user-images.githubusercontent.com/60064066/109001139-d9757600-76a4-11eb-8f3d-5c97c718b37a.png)
